### ISSUE TITLE
Option to use single click to navigate through filmstrip in darkroom

### ISSUE THREAD

**Issue by SammysHP** [darktable-org/darktable#18107]
**Is your feature request related to a problem? Please describe.**

With v5.0 the behavior of navigation in the filmstrip was changed. Previously you could jump to another image by a single click on an image while in darkroom mode. Now you need to double click, while a single click just changes the selection (but how does it make sense to select multiple images in darkroom mode?).

While editing, I jump back and forth a lot, often skipping images. Thus a simple "prev/next image" shortcut isn't sufficient in my case. The change in v5.0 severely impacted my editing performance, without any (obvious) benefit.

**Describe the solution you'd like**

I'd like to see an option to bring back the old behavior of changing the current image with a single click in the filmstrip while in darkroom mode. The filmstrip in darkroom mode is special and having a different behavior here does not affect UI consistency.

In my opinion it also does not make sense to show the current selection in the filmstrip while in darkroom mode. Maybe I have been using Darktable wrong for all those years, but the focus in darkroom is on editing a single image at a time, not collection management.

**Comment by ralfbrown** [comment]
It's unlikely to be changed, as the change in 5.0 restores old behavior and harmonizes the behavior with the lighttable thumbnail grid and basically every file manager ever.

Selecting images in the filmstrip enables cut&paste of edit histories and the new "synchronize" shortcut on Ctrl-X, as well as no-click star ratings and color marking.

**Comment by SammysHP** [comment]
> Selecting images in the filmstrip enables cut&paste of edit histories and the new "synchronize" shortcut on Ctrl-X, as well as no-click star ratings and color marking.

Ahh I see, these are all things I do from the lighttable for better overview. But I assume there are many different valid workflows.

> It's unlikely to be changed, as the change in 5.0 restores old behavior and harmonizes the behavior with the lighttable thumbnail grid and basically every file manager ever.

I can't remember that it ever was a double click. As I said, I don't see any necessity to have this kind of consistency between lighttable and darkroom as they have different purposes/focus (for me, that is).

Regarding other file managers: This isn't a file manager. It's a navigation method while editing images. For example gThumb has it implemented as I suggested (double click from the lighttable, but single click from the filmstrip).

Also I don't request to change the default behavior, just to add an option to get the old behavior back. Now that v5 is rolling out, I predict that more long time users are confused about this change.

**Comment by SammysHP** [comment]
> so I don't want to see a pref for this as it will require more testing and maintenance.

Understandable, that's something I have to deal with as well as a software developer. It's just hard for me to see the benefit of the change *specifically for the darkroom layout* because I haven't found any use case or example of a workflow. Even the [documentation](https://docs.darktable.org/usermanual/development/en/module-reference/utility-modules/shared/filmstrip/) appears to be outdated and not helpful in this regard.

**Comment by SammysHP** [comment]
Yeah, probably. Still I'm curious to learn more about the new possibilities. Can you link me any resources how to work with the filmstrip in the darkroom layout to utilize the selection feature?

**Comment by SammysHP** [comment]
Yes, I'm familiar with selecting multiple images. But nobody was able to tell me what to do with the selection in the darkroom layout.
